src/contract.ts

```diff
@@ -40,7 +47,10 @@ export class Contract {
                 writable: false,
                 enumerable: true,
                 value: nameFunction(methodName, async (args: object = {}, gas?: BN, amount?: BN, ...ignored) => {
-                    if (ignored.length || Object.prototype.toString.call(args) !== '[object Object]') {
+                    if (
```

Comment:
nit: this formatting of `if (` almost gives me physical pain lol

Reply:
hahaha, ok, I'll find a way to shorten it enough to fit on one line comfortably